core/server/master/src/main/java/alluxio/master/file/InodeSyncStream.java

```diff
@@ -789,8 +790,19 @@ private void loadMetadata(LockedInodePath inodePath, LoadMetadataContext context
               LOG.debug("Failed to loadMetadata because file is not in ufs:"
                       + " inodePath={}, options={}.",
                   childURI, loadMetadataContext, e);
+            } catch (BlockInfoException | FileAlreadyCompletedException
+                | FileDoesNotExistException | InvalidFileSizeException
+                | IOException e) {
+              LOG.debug("Failed to loadMetadata because the ufs file or directory"
+                      + " is {}, options={}.",
+                      childStatus, loadMetadataContext, e);
```

Comment:
Line 797 and 798 should have 4 more spaces from line 796 instead of 8. The checkstyle currently is not able to tell the incorrect indentation, unfortunately.